pkg/actor/decommission_test.go

```diff
@@ -30,6 +30,7 @@ import (
 func TestDecommisionFeatureFlag(t *testing.T) {
 	// FeatureGate is currently disabled
 	assert.False(t, utilfeature.DefaultMutableFeatureGate.Enabled(features.Decommission))
+	assert.False(t, utilfeature.DefaultMutableFeatureGate.Enabled(features.AutoPrunePVC))
```

Comment:
@udnay here is the test that the feature gate is shut off. PTAL so we can merge

Reply:
Sure, lets merge this but I also was looking for a test to make sure that when `AutoPrunePVC` is `false` we never call the pruner. This is fine for now though.